mllib/src/main/scala/org/apache/spark/ml/classification/LogisticRegression.scala

```diff
@@ -295,6 +295,13 @@ class LogisticRegression @Since("1.2.0") (
     instr.logParams(regParam, elasticNetParam, standardization, threshold,
       maxIter, tol, fitIntercept)
 
+    val autoAggregationDepth =
+      if ($(aggregationDepth) > 0) $(aggregationDepth)
+      else getAggregationDepthByFormula(
+        instances.context.getConf.getSizeAsBytes("spark.driver.memory", "1g"),
```

Comment:
yeah, but how to get the default `1g` config ? I seems not found the pre-defined const val or something like that ? or is there a method such as `getOrDefault`?